src/Aquifer.API/Endpoints/Bibles/Book/List/Endpoint.cs

```diff
@@ -16,23 +19,41 @@ public override void Configure()
 
     public override async Task HandleAsync(Request req, CancellationToken ct)
     {
-        var response = await dbContext.BibleBookContents
-            .Where(bbc => bbc.Bible.Enabled && bbc.BibleId == req.BibleId)
-            .Select(bbc => new Response
-            {
-                Code = bbc.Book.Code,
-                Number = (int)bbc.BookId,
-                LocalizedName = bbc.DisplayName,
-                TotalChapters = bbc.ChapterCount,
+        var maxChapterNumberAndBookendVerseNumbersByBookIdMap =
+            await _cachingVersificationService.GetMaxChapterNumberAndBookendVerseNumbersByBookIdMapAsync(req.BibleId, ct);
 
-                // TODO TotalVerses here should come from actual Bible data, not the standard BookChapters English versification data.
-                // After this usage of Book.Chapters is removed we should also delete that association altogether.
-                Chapters = bbc.Book.Chapters.OrderBy(c => c.Number).Select(c => new ResponseChapter
+        var response = (await _dbContext.BibleBookContents
+                .Where(bbc => bbc.Bible.Enabled && bbc.BibleId == req.BibleId)
+                .Select(
+                    bbc => new
+                    {
+                        bbc.BookId,
+                        bbc.Book.Code,
+                        bbc.DisplayName,
+                        bbc.ChapterCount,
+                    })
+                .ToListAsync(ct))
+            .Select(
+                x => new Response
                 {
-                    Number = c.Number,
-                    TotalVerses = c.MaxVerseNumber
+                    Code = x.Code,
+                    Number = (int)x.BookId,
+                    LocalizedName = x.DisplayName,
+                    TotalChapters = x.ChapterCount,
+                    Chapters = maxChapterNumberAndBookendVerseNumbersByBookIdMap
+                            .GetValueOrNull(x.BookId)
+                            ?.BookendVerseNumbersByChapterNumberMap
+                            .OrderBy(kvp => kvp.Key)
+                            .Select(
+                                kvp => new ResponseChapter
+                                {
+                                    Number = kvp.Key,
+                                    TotalVerses = kvp.Value.MaxVerseNumber,
```

Comment:
I'm thinking we might need to add `MinVerseNumber` here and update Well so that it displays verse 0 in the picker.  And probably rename `TotalVerses` to `MaxVerseNumber` while we're at it.